rgw/v2/tests/s3_swift/test_bucket_notifications.py

```diff
@@ -133,7 +133,7 @@ def test_exec(config, ssh_con):
         rgw_admin_notif_commands_available = False
         ceph_version_id = ceph_version_id.split("-")
         ceph_version_id = ceph_version_id[0].split(".")
-        if (
+        if (float(ceph_version_id[0]) >= 19) or (
             float(ceph_version_id[0]) == 18
```

Comment:
will "float(ceph_version_id[0]) >= 18" not work?

Reply:
@anrao19 version > 7.1 should work

Reply:
yes, so can we update script as "**if float(ceph_version_id[0]) >= 18 and ...**) instead of 
**if (float(ceph_version_id[0]) >= 19) or (
            float(ceph_version_id[0]) == 18** and ...

Reply:
@anrao19 but this fails on 8.0 if we modify conditional like this
`if float(ceph_version_id[0]) >= 18 and float(ceph_version_id[1]) >= 2 and float(ceph_version_id[2]) >= 1)`
because the ceph 8.0 version is 19.1.0-42.el9cp

the version should be greater than 18.2.1 or greater than or equal to 19